docs-partials/builder/vsphere/clone/CustomizeConfig-not-required.mdx

```diff
@@ -1,11 +1,13 @@
-<!-- Code generated from the comments of the CustomizeConfig struct in builder/vsphere/clone/step_customize.go; DO NOT EDIT MANUALLY -->
+<!-- Code generated from the comments of the CustomizeConfig struct in builder\vsphere\clone\step_customize.go; DO NOT EDIT MANUALLY -->
```

Comment:
looks like packer-sdc needs to generate forward slashes even when executed on windows.

Reply:
Issue with packer-sdc raised here: https://github.com/hashicorp/packer-plugin-sdk/issues/104
Note: for this pull requerst, I have manually edited the comments as a workaround until this issue is solved.

Reply:
The fix for packer-sdc on windows was merged.